pkg/test/kds/setup/server.go

```diff
@@ -52,7 +52,7 @@ func StartServer(store store.ResourceStore, wg *sync.WaitGroup, clusterID string
 		cfg:     kuma_cp.Config{},
 		metrics: metrics,
 	}
-	srv, err := kds_server.New(core.Log, rt, providedTypes, clusterID, 100*time.Millisecond, providedFilter, false)
+	srv, err := kds_server.New(core.Log.WithName("kds-"+clusterID), rt, providedTypes, clusterID, 100*time.Millisecond, providedFilter, false)
```

Comment:
how about `WithName("kds").WithValues("clusterID", clusterID)`? If the log is "dynamic" then our pattern is to put this in `WithValues()`.

Reply:
yeh `WithName` sounds better.